capsul/process/process.py

```diff
@@ -736,10 +736,8 @@ def set_parameter(self, name, value):
         value: object (mandatory)
             the trait value we want to set
         """
-        # Detect File and Directory trait types with None value
-        if value is None and is_trait_pathname(self.trait(name)):
-
-            # The None trait value is _Undefined, do the replacement
+        # The None trait value is _Undefined, do the replacement
```

Comment:
This comment is related to my description language comment. In addition here I see only potential issues with the Any trait. But the question is: do we accept 'None' values in the pipeline descriptions?